src/main/java/com/graphhopper/converter/api/GisgraphySearchEntry.java

```diff
@@ -0,0 +1,222 @@
+			add("PL");
+			add("RU");
+			add("SI");
+			add("SK");
+			add("SW");
+			add("TR");
+		}
+	};
+
+
+	@JsonProperty("feature_id")
+	private long featureId;
+	
+	private double lat;
+	
+	private double lng;
+	
+	private String name;
+
+	@JsonProperty("country_code")
+	private String countryCode;
+
+	@JsonProperty("is_in")
+	private String isIn;
+
+	@JsonProperty("is_in_place")
+	private String isInPlace;
+
+	@JsonProperty("is_in_zip")
+	private List<String> isInZip;
+
+	@JsonProperty("adm1_name")
+	private String adm1Name;
+
+	@JsonProperty("zip_code")
+	private List<String> zipCodes;
+
+	@JsonProperty("house_numbers")
+	private List<String> houseNumbers = new ArrayList<String>();
+
+	public List<String> getHouseNumbers() {
+		return houseNumbers;
+	}
+
+	public void setHouseNumbers(List<String> houseNumbers) {
+		this.houseNumbers = houseNumbers;
+	}
+
+	@JsonProperty("house_number")
+	private String houseNumber;
+
+
+
+	public long getFeatureId() {
+		return featureId;
+	}
+
+	public void setFeatureId(long featureId) {
+		this.featureId = featureId;
+	}
+
+	public double getLat() {
+		return lat;
+	}
+
+	public void setLat(double lat) {
+		this.lat = lat;
+	}
+	public double getLng() {
+		return lng;
+	}
+
+	public void setLng(double lng) {
+		this.lng = lng;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getCountryCode() {
+		return countryCode;
+	}
+
+	public void setCountryCode(String countryCode) {
+		this.countryCode = countryCode;
+	}
+
+	public String getIsIn() {
+		return isIn;
+	}
+
+	public void setIsIn(String isIn) {
+		this.isIn = isIn;
+	}
+	@JsonProperty("is_in_place")
+	public String getIsInPlace() {
+		return isInPlace;
+	}
+
+	@JsonProperty("is_in_place")
+	public void setIsInPlace(String isInPlace) {
+		this.isInPlace = isInPlace;
+	}
+
+	@JsonProperty("is_in_zip")
+	public List<String> getIsInZip() {
+		return isInZip;
+	}
+
+	@JsonProperty("is_in_zip")
+	public void setIsInZip(List<String> isInZip) {
+		this.isInZip = isInZip;
+	}
+
+	public String getAdm1Name() {
+		return adm1Name;
+	}
+
+	public void setAdm1Name(String adm1Name) {
+		this.adm1Name = adm1Name;
+	}
+
+	public List<String> getZipCodes() {
+		return zipCodes;
+	}
+
+	public void setZipCodes(List<String> zipCodes) {
+		this.zipCodes = zipCodes;
+	}
+
+
+	public String getCountry() {
+		if (countryCode!=null){
+			return CountryInfo.countryLookupMap.get(countryCode.toUpperCase());
+		}
+		return null;
+	}
+
+
+	@JsonProperty("house_number")
+	public String getHouseNumber() {
+		return houseNumber;
+	}
+
+	@JsonProperty("house_number")
+	public void setHouseNumber(String houseNumber) {
+		this.houseNumber = houseNumber;
+	}
+
+	public String getZipCode() {
+		if (zipCodes!=null && zipCodes.size()>0){
+			return zipCodes.get(0);
+		}
+		return null;
+	}
+
+	@JsonProperty("label")
+	public String getLabel() {
+		StringBuffer addressFormated = new StringBuffer();
```

Comment:
Would it make more sense to use `formatedFull` instead?

Reply:
no formatedFull won't have the house number. Gisgraphy search for street and a street with house number is return. we don't store address in SolR but streets (it allow address interpolation and more)

Reply:
Could you explain this a bit more? I tried to get a real live example and couldn't find any house numbers via gisgraphy?

I tried for example `Universitätsbibliothek, Standort Stadtmitte`, [in OSM it contains a housenumber](http://www.openstreetmap.org/relation/1553281). When I looked it up via [Gisgraphy fulltext](https://services.gisgraphy.com/fulltext/?q=Universit%C3%A4tsbibliothek,%20Standort%20Stadtmitte&style=full), there is no house number. That doesn't trouble me, but I would like to avoid business logic in the geocoding-converter, if it's avoidable. The simpler the better :). This also means we don't have to update the geocoding-converter if any geocoding provider changes details about their implementation.

That's why I was asking about using `formattedFull` or for fulltext it seems to be `fully_qualified_name`.